client/components/common/Functions.tsx

```diff
@@ -0,0 +1,3 @@
+export const capitalizeFirstLetter = (str: string): string => {
```

Comment:
`/common/Functions` is somewhat of a weird name. Something like `/common/utils` might be better?